src/components/seconardynavnew/desktop.tsx

```diff
@@ -33,6 +33,7 @@ const StyledSecondaryLinks = {
             theme.space.inc90,
         ],
     },
+    whiteSpace: 'nowrap' as 'nowrap',
```

Comment:
Is the `'nowrap' as 'nowrap'` needed here?

Just realized "seconardynavnew/desktop.tsx"  :-)

Reply:
It is when not using an object literal in the sx prop, I think because it sees `StyledSecondaryLinks` as a plain object, but if you pass an object literal as the prop it infers it as a `ThemeUIStyleObject`. The proper way to do this is actually probably to define it as a `ThemeUIStyleObject`: 
```typescript
const StyledSecondaryLinks: ThemeUIStyleObject = {
    ...,
    whiteSpace: 'nowrap',
};
```
But this pattern is everywhere so it might be worth just making a ticket to clean that up everywhere.